src/CXml/Model/Request/PunchOutSetupRequest.php

```diff
@@ -22,6 +24,14 @@ class PunchOutSetupRequest implements RequestPayloadInterface
 	 */
 	private string $buyerCookie;
 
+	/**
+	 * @Ser\XmlList(inline=true, entry="Extrinsic")
+	 * @Ser\Type("array<CXml\Model\Extrinsic>")
+	 *
+	 * @var Extrinsic[]
+	 */
+	protected array $extrinsics = [];
```

Comment:
@mathielen This is important here to respect the order in serialize, otherwise the extrinsics nodes are rendered at the end and then the DTD Validation failed.

Reply:
@mathielen,

Extrinsics elements are added last in the PunchOutSetupRequest. The Dtd validator throws the following errors:
`CXml\Validation\Exception\CXmlInvalidException: Element PunchOutSetupRequest content does not follow the DTD, expecting (BuyerCookie , Extrinsic* , BrowserFormPost? , Contact* , SupplierSetup? , ShipTo? , SelectedItem? , ItemOut*), got (BuyerCookie BrowserFormPost SupplierSetup ShipTo SelectedItem ItemOut ItemOut Extrinsic ) at line 23, column 0. Code 504.`


According to DTD "Extrinsics" should be after "BuyerCookie". 
Is there another way to set the order of elements?

Reply:
Oh right, true. Well then, lets do it this way!